src/main/java/org/jabref/gui/mergeentries/FetchAndMergeEntry.java

```diff
@@ -76,7 +76,17 @@ public void fetchAndMerge(BibEntry entry, List<Field> fields) {
                                   })
                                   .onFailure(exception -> {
                                       LOGGER.error("Error while fetching bibliographic information", exception);
-                                      dialogService.showErrorDialogAndWait(exception);
+                                      String localMessage = exception.getLocalizedMessage();
+                                      // client error
+                                      if (localMessage.startsWith("Client")) {
```

Comment:
This is the check, I am talking about. Personally, I think this should not be a string comparison, but a different, more robust check.

I assume, that @Siedlerchr had something similarly in mind with his remark.